app/lib/common/gateways.js

```diff
@@ -35,7 +35,11 @@ export const availableGateways = {
         isSimple: true,
         selected: false,
         simpleAssetGateway: true,
-        fixedMemo: {prepend: "dex:", append: ""},
+        fixedMemo: {
```

Comment:
does the change from prepend to prepend_default prepend_btsid break anything for other gateway integrations? backwards compatible?

Reply:
Yes backward compatibility is provided.
At the Rudex gateway, the memo of both the old and the new format is currently correctly processed and compatibility will continue to be maintained.

In the frontend at the moment, this option (fixedMemo) is used only by Rudex and this cann't break the work of other gateways.

Reply:
> does the change from prepend to prepend_default prepend_btsid break anything for other gateway integrations? backwards compatible?

Yes backward compatibility is provided.
At the Rudex gateway, the memo of both the old and the new format is currently correctly processed and compatibility will continue to be maintained.

In the frontend at the moment, this option (fixedMemo) is used only by Rudex and this cann't break the work of other gateways.